pinot-plugins/pinot-input-format/pinot-json/pom.xml

```diff
@@ -39,5 +39,11 @@
       <groupId>commons-lang</groupId>
       <artifactId>commons-lang</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.pinot</groupId>
+      <artifactId>pinot-spi</artifactId>
```

Comment:
is this needed?

Reply:
The build failed with a dependency error without it.

Reply:
I thought `pinot-spi` is in `pinot-plugins` pom (https://github.com/apache/incubator-pinot/blob/master/pinot-plugins/pom.xml#L50) so you don't need to put it here.
Maybe to rebase from current master?